packages/react-router-config/jest.config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  testRunner: "jest-circus/runner",
+  restoreMocks: true,
```

Comment:
`restoreMocks` will restore the implementation of any spies after every test, here this keeps us from needing to do `console.error.mockResore` at the end of every test that spies on `console.log`